### PR TITLE
Fixed CT, USA scraper

### PR DESCRIPTION
**Pull request by chunder**

### CODE REVIEW

File: src/events/crawler/scrapers/USA/CT/index.js [hunk 1]
```diff
@@ -99,8 +99,20 @@ const scraper = {
         if (!Number.isNaN(parse.number(rows[i][0]))) continue;
 
         const countyName = geography.addCounty(rows[i][0]);
-        const cases = parse.number(rows[i][1]);
-        const deaths = parse.number(rows[i][3]);
+
+        let cases = null;
+        let deaths = null;
```
Comment (qgolsteyn) [inline src/events/crawler/scrapers/USA/CT/index.js]: Should set `cases` or `deaths` to undefined if we don't have a value for them?

Comment (chunder) [reply src/events/crawler/scrapers/USA/CT/index.js]: The code will either give them a value or throw an error.

Comment (chunder) [reply src/events/crawler/scrapers/USA/CT/index.js]: Also, the linter won't let me initialize them to undefined!

**Comment by chunder** [comment]
Ok, no more null.

**Pull request merged**